### PR TITLE
Use valid color value for banner background

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: docs/source/_static/main-page.css [hunk 1]
```diff
@@ -44,7 +44,7 @@ body {
 }
 
 .hero {
-  background-color: #000000e0;
```
Comment (mrocklin) [inline docs/source/_static/main-page.css]: OK, this is interesting.  We actually want this to be slightly transparent, so the `E0` value at the end is important.  Perhaps your browser didn't like the lowercased 'e'?  Does this change fix the situation in your case?

Comment (mrocklin) [reply docs/source/_static/main-page.css]: We could also use `#202020` instead.  Does that work better in your browser @stefanv ?  This is something that you can probably set manually using your browser's dev tools to see how it looks.  Happy to walk you through this if you're unfamiliar.

Comment (1354092549) [reply docs/source/_static/main-page.css]: @mrocklin 
This is invalid color value, please try `background-color: rgba(0,0,0,0.8);`

Comment (stefanv) [reply docs/source/_static/main-page.css]: That's the correct solution, I think.

Comment (stefanv) [reply docs/source/_static/main-page.css]: (PR updated)

**Comment by stefanv** [comment]
@mrocklin Aside: you may want to compile the docs with the newer version of numpydoc that just came out.  On my system I see a bunch of warnings, that the old version may not have raised.

**Comment by mrocklin** [comment]
@stefanv thanks for the pointer about numpydoc.  Updated here: https://github.com/dask/dask/pull/3380

@1354092549 thanks for the information about colors.  I'm confused. I thought that hex color values of the form `#RRGGBBAA` were valid.  Is this only true on certain browsers?  Is this not actually true?  Any references here would be welcome.

**Comment by 1354092549** [comment]
https://www.w3.org/TR/css-color-3/#rgba-color

**Pull request merged**